actors/builtin/paych/paych_actor.go

```diff
@@ -64,9 +65,9 @@ func (pca *Actor) Constructor(rt vmr.Runtime, params *ConstructorParams) *adt.Em
 // The account actor constructor checks that the embedded address is associated with an appropriate key.
 // An alternative (more expensive) would be to send a message to the actor to fetch its key.
 func (pca *Actor) resolveAccount(rt vmr.Runtime, raw addr.Address) (addr.Address, error) {
-	resolved, ok := rt.ResolveAddress(raw)
-	if !ok {
-		return addr.Undef, exitcode.ErrNotFound.Wrapf("failed to resolve address %v", raw)
+	resolved, err := builtin.ResolveToIDAddr(rt, raw)
+	if err != nil {
+		return addr.Undef, xerrors.Errorf("failed to resolve address %v: %w", raw, err)
```

Comment:
`RequireNoError`

Reply:
Have changed this function to return the exit code to the caller.